Tasks/app-store-promote/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -5,8 +5,8 @@
   "loc.instanceNameFormat": "Submit to the App Store for review",
   "loc.group.displayName.advanced": "Advanced",
   "loc.input.label.authType": "Authentication Method",
-  "loc.input.label.serviceEndpoint": "Service Endpoint",
-  "loc.input.help.serviceEndpoint": "The TFS/Team Services service endpoint that specifies the identity used to publish the app.",
+  "loc.input.label.serviceEndpoint": "Service Connection",
+  "loc.input.help.serviceEndpoint": "The TFS/Azure DevOps service connection that specifies the identity used to publish the app.",
```

Comment:
Do we even want to use TFS anymore for naming?

Reply:
I left it in there since we still support TFS versions that we already shipped.

Reply:
Ah, didn't see this.  In that case, please ignore all comments suggesting otherwise! :)